client/web/package.json

```diff
@@ -16,6 +16,8 @@
 	"license": "Apache-2.0",
 	"homepage": "https://www.graphite.design",
 	"dependencies": {
+		"lol": "github:wk-js/lol.js",
+		"simple-color-picker": "^1.0.4",
```

Comment:
The goal for Graphite is to avoid dependencies unless they are of significant importance, so let's aim to remove our dependence of these before merging. I would be happy to write the color picker component for you if you are concerned about losing the convenience provided by `simple-color-picker`.